README.md

```diff
@@ -187,6 +188,16 @@ $ lukso update
 
 ```
 
+### `update configs`
+`update configs` Update chain configuration files. This commands overwrites your oldchain configs, but keeps your client configs.
+
+#### How to update chain config files
+```sh
+# starts an update of chain config files. 
+$ lukso update configs
```

Comment:
Please mention about old Prysm <=4.0.8 possible error logs.

Reply:
We have possible errors mentioned in https://github.com/lukso-network/tools-lukso-cli/pull/232 - I don't think that this command is a good place to mention this - if we want to provide this info to every prysm related place, we would have to mention it in every other command. If we want to add some kind of information about those error logs in a one, most relevant command, I think the best one would be `start` - because Prysm throws errors when ran, using `start`.

Requirements to update configs were added :+1:

Reply:
Ok, let's document under `start` command then

Reply:
Added in #232